src/esp/bindings/SensorBindings.cpp

```diff
@@ -112,7 +112,26 @@ void initSensorBindings(py::module& m) {
              VisualSensor, Magnum::SceneGraph::PyFeatureHolder<CameraSensor>>(
       m, "CameraSensor")
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
-                          const SensorSpec::ptr&>());
+                          const SensorSpec::ptr&>())
+      .def("set_projection_params", &CameraSensor::setProjectionParameters,
```

Comment:
Is this really just setting the projection_params? Maybe a better API would be reconfigure_with_params or something? A more informative API here would be helpful.

Reply:
Yes, what it does is take the passed sensor spec and copy the projection-related params into the sensor's spec.  Then, everything relating to those params is rebuilt.